src/Files.App/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -376,7 +392,7 @@ private static LayoutPreferences GetDefaultLayoutPreferences(string folderPath)
 
 			if (folderPath == CommonPaths.DownloadsPath)
 				// Default for downloads folder is to group by date created
-				return new LayoutPreferences() { DirectoryGroupOption = GroupOption.DateCreated };
+				return new LayoutPreferences() { DirectoryGroupOption = GroupOption.DateCreated, DirectoryGroupDirection = SortDirection.Descending };
```

Comment:
Can you split this on to separate lines?

Reply:
Done.